src/zb_links/wsgi.py

```diff
@@ -1,3 +1,17 @@
-from zb_links.app import create_app
+from zb_links import config
+from zb_links.app import create_app as create_api_app
+from zb_links.db.init_tables.init_db_app import create_app as create_db_app
 
-application = create_app()
+
+application = None
+try:
+    flask_app = config["application"]["FLASK_APP"]
```

Comment:
Why is this inside the try catch block? Is this the key, error to catch? @fmux suggested to me in a different context that
```python
value=b.get("key", "default")
```
is one alternative to
```python
value = None
try:
  value = b["key"]
except KeyError:
 value = "default"
```
I am not a python expert, but I would imagine that the .get method is easier to read for most people - at lest for this simple case - regardless of [EAFP](https://docs.python.org/2/glossary.html#term-eafp)

Reply:
b.get("key", "default") is equivalent to b['key']['default']

I think Fabian includes default values in config files, and that could be used here, e.g.
[application]
default_app = zblinks.app.py
flask_app = ...

and then the user would only add flask_app value if some other app is used (for instance the app for migrations and seeding)

this could help a bit in the if statements in the next comment.

Reply:
With the get-method, I was referring to https://docs.python.org/3/library/stdtypes.html#dict.get
Anyhow, even with the try - except structure, I would limit the block to the line where you expect the key error. Otherwise, you might face unexpected behavior, if a key-error occurs in other statements of that code block.

Reply:
> b.get("key", "default") is equivalent to b['key']['default']
I know this is already merged, but just to make it clear here: This statement is not correct. `value = b.get("key", "default")` is equivalent to
```python
if "key" in b:
    value = b["key"]
else:
    value = "default"`
```
(or to the try/except version that @physikerwelt already wrote). In particular, `"default"` is used as a value, not as a key of `b["key"]`!